### DISCUSSION TITLE
Opensource WebUI of the old app

### DISCUSSION THREAD

**Discussion by dev-rvk** [yume-chan/ya-webadb#657] [General]
Hi,
There are two UI's that are deployed, Can I get the source code for the UI's?
I know that the new UI is closed source, but is the old UI that is deployed open?
[https://app.webadb.com/](https://app.webadb.com/)
[https://ya-webadb.vercel.app/](https://ya-webadb.vercel.app/)

**Comment by yume-chan** [comment]
Yes: https://github.com/tango-adb/old-demo

> https://app.webadb.com/

This one is not mine. Someone modified my code and deployed there. (and that's one of the reasons why the new UI is not open sourced)

**Reply by dev-rvk** [reply]
Thanks a lot!
